README.md

```diff
@@ -2,9 +2,13 @@
 De documentatie is nog volop in ontwikkeling
 ```
 
-# Kennisbank
+## OSPO-NL Kennisbank
```

Comment:
Deze is standaard een 'Heading 1' ... dus één `#`. Wat denk jij? Verder helemaal goed zo!